galley/pkg/server/callout.go

```diff
@@ -25,8 +25,11 @@ import (
 	mcp "istio.io/api/mcp/v1alpha1"
 	"istio.io/istio/galley/pkg/authplugins"
 	"istio.io/istio/pkg/mcp/source"
+	"istio.io/istio/pkg/version"
 )
 
+const versionMetadataKey = "galley-version"
```

Comment:
I don't think "galley-version" is the right name for this. Call it config.source.version?

Reply:
It's the version that gets baked into galley when the binary is built. I don't want to call it "istio version" as it doesn't necessarily represent the version of the whole mesh. If we consider "config source" an analog for galley, then that works for me.